news/views.py

```
@@ -561,10 +576,58 @@ def post(self, request, *args, **kwargs):
+                        lib = Library.objects.get(slug=related_library)
+                    except Library.DoesNotExist:
+                        messages.error(
+                            request,
+                            _(
+                                "That related library does not exist, please select another."
+                            ),
+                        )
+                        context = self.get_context_data()
+                        return self.render_to_response(context)
+
+                    tag, created = ContentTag.objects.get_or_create(
```

Ruff (0.15.21)

[warning] Unpacked variable `created` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)